src/App.vue

```diff
@@ -1,28 +1,28 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <Timeline></Timeline>
+    <EventEditor></EventEditor>
```

Comment:
If you don't have slotted content to pass to the component, then the component tag can be self-closing:

```html
<Timeline />
<EventEditor />
```
It's a best practice in the Vue world. Check out how you can force that convention in your eslint config file:

https://eslint.vuejs.org/rules/html-self-closing.html#vue-html-self-closing